src/components/breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <nav :class="breadcrumbClasses">
+        <ul>
+            <template v-for="(item, index) in items">
+                <li
+                    v-if="item"
+                    :key="item"
+                    :class="{ 'is-active': index === items.length - 1 }">
+                    <component
+                        :is="tag"
+                        :href="tag == 'a' ? item.path : undefined"
```

Comment:
What do you think to create a `breadcrumb-item` component that contains `tag` and all other props applied on root element like `navbar-item`

Reply:
I Have tried that before but I didn't knew that you want to keep same standards between the components!
However I'm gonna Implement it.

Reply:
Thanks

Reply:
@jtommy I'm done!